source/results/flux_points/index.rst

```diff
@@ -80,6 +78,33 @@ error and upper limit columns are:
 * ``flux_ul`` : Upper limit with confidence level given by ``UL_CONF``
   header keyword.
 
+A row may have a value and any combination of upper limits and errors:
+
+.. code::
```

Comment:
I think it should be
```
.. code-block:: none
```
instead of
```
.. code::
```
(see http://www.sphinx-doc.org/en/latest/markup/code.html)

(used three times, I'll just leave one comment)